docs/user-guide/admin-cli.md

```diff
@@ -87,45 +87,106 @@ optional arguments:
 
 ### Create
 A vmgroup named "_DEFAULT" will be created automatically post install.
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                Default_datastore  VM_list
+------------------------------------  --------  -------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup  __VM_DS
+```
+
+The "Default_datastore" field is set to "__VM_DS" for "_DEFAULT" vmgroup. Any volume create from VM which belongs to "_DEFAULT" vmgroup will be created on the datastore where VM resides.
+
+Two full access privileges will be created automatically post install.
```

Comment:
This line is hard to understand. Better to say "When configuration is initialized with 'config init',  the access to __ALL_DS and __VM_DS for all VMs is automatically enabled"